homeassistant/components/recorder/db_schema.py

```diff
@@ -344,20 +368,26 @@ class States(Base):
     attributes_id: Mapped[int | None] = mapped_column(
         Integer, ForeignKey("state_attributes.attributes_id"), index=True
     )
-    context_id: Mapped[str | None] = mapped_column(
+    context_id: Mapped[str | None] = mapped_column(  # no longer used
         String(MAX_LENGTH_EVENT_CONTEXT_ID), index=True
```

Comment:
Need to disable the index here and let the defined table index with the MySQL length be used instead